File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
N = int(input())
arr = list(map(int, input().split()))

cnt = 0

for num in arr:
    if num != 1:
        check = True
        for _ in range(2, num):
            if num%_ == 0:
                check = False
                break
        if check:
            cnt += 1

print(cnt)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.cc
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int N, cnt=0;
    scanf("%d", &N);
    for (int i=0; i<N; i++) {
        int n;
        scanf("%d", &n);
        if (n == 1) continue;
        bool check = true;
        for (int j=2; j< (int)sqrt(n) + 1; j++) {
            if (n%j == 0) {
                check = false;
                break;
            }
        }
        if (check) cnt++;
    }
    cout << cnt;
}